DependencyInjection/Configuration.php

```diff
@@ -90,6 +90,13 @@ public function getConfigTreeBuilder()
                     ->canBeEnabled()
                     ->children()
                         ->scalarNode('default_chan_frequency')->defaultValue('always')->end()
+                        ->arrayNode('templates')
+                        ->addDefaultsIfNotSet()
+                            ->children()
+                                ->scalarNode('xml')->defaultValue('CmfSeoBundle:Sitemap:index.xml.twig')->end()
```

Comment:
i think this should be an array with the format as key, so that users can add even keys that the bundle knows nothing about. the controller is built that way. the format should look like this: https://github.com/symfony-cmf/RoutingBundle/blob/master/DependencyInjection/Configuration.php#L41-L45 but with useAttributeAsKey('format')

not sure if we can put the defaults in the Configuration class in this case - maybe something like ` ->beforeNormalization()->ifEmpty()->then(function ($v) { return array('xml' => ...); })->end()` (did not test this so might not work exactly like this)